tests/fixture/tmpnet/process_runtime.go

```diff
@@ -211,7 +211,7 @@ func (p *ProcessRuntime) waitForProcessContext(ctx context.Context) error {
 	ctx, cancel := context.WithTimeout(ctx, defaultNodeInitTimeout)
 	defer cancel()
 	for len(p.node.URI) == 0 {
-		err := p.readState()
+		err := p.readState(ctx)
```

Comment:
pass in `context.Background()` to avoid any confusion, given it's unused by `readState` (or even `nil`, in case the context gets used in the future, to produce a panic)

```suggestion
		err := p.readState(nil)
```